css/styles.css

```diff
@@ -1,20 +1,20 @@
 .header-container {
-    background-image: url(../image/header-image.jpg);
-    flex-wrap: wrap;
-    align-items: center;
-    max-width: 340px;
-    height: 620px;
-    margin: 0 auto 10vh auto;
+  background-image: url(../image/header-image.jpg);
+  flex-wrap: wrap;
+  align-items: center;
+  max-width: 340px;
+  height: 620px;
+  margin: 0 auto 10vh auto;
 }

```

Comment:
- [ ] [REQUIRED] Good job so far! The design in `Headline` section of your project is not exactly like [Figma design](https://www.figma.com/files/recent?fuid=1045426358203651467), please correct them.

![image](https://user-images.githubusercontent.com/63915024/176981068-2d60dc51-654b-4ebf-a193-86d6f9c7543b.png)